### ISSUE TITLE
Smelting Stone Bricks into Smooth Stone Bricks conflicts with the vanilla Cracked Stone Brick smelting recipe.

### ISSUE THREAD

**Issue by Vaelzan** [elucent/eidolon#34]
**Describe the bug**
Eidolon adds a recipe for Smooth Stone Bricks by smelting Stone Bricks, but this conflicts with the vanilla Minecraft recipe to make Cracked Stone Bricks. With a mod like Polymorph installed this can be worked around, but for many players this will simply make Cracked Stone Bricks impossible to craft.

A proposed solution would be to simply remove this recipe - the stonecutter is probably enough.

**To Reproduce**
Smelt vanilla stone bricks in a furnace or other smelting method. 
Expected result is Cracked Stone Bricks, but actual result is Smooth Stone Bricks.

**Environment**
Eidolon 0.2.7 (still present in latest GitHub source)
Forge 35.1.28

**Crash Report**
N/A

**Comment by elucent** [comment]
Yeah, I'll just remove the smelting recipe. Thanks for reminding me. :)

**Comment by elucent** [comment]
Will be resolved in release 0.2.8.
